examples/simple/library.cc

```diff
@@ -0,0 +1,30 @@
+#include "opentelemetry/trace/provider.h"
+
+namespace trace = opentelemetry::trace;
+namespace nostd = opentelemetry::nostd;
+
+static nostd::shared_ptr<trace::Tracer> get_tracer()
+{
+  auto provider = trace::Provider::GetTracerProvider();
+  return provider->GetTracer("");
```

Comment:
People will copy-paste from examples. Please "" -> "foo_library" or whatever the arg should be. :)

Reply:
Good call. Done.